### PR TITLE
[PHILLY-2017] Initial PR, hold the date.

### PR DESCRIPTION
**Pull request by peterjshan**
On the tin.

cc: @bridgetkromhout

### CODE REVIEW

**Comment by MikeRosTX** [comment]
@peterjshan make sure you're using the latest version of Hugo, because there are errors that are not allowing us to merge your PR.  Make sure that you can test to see if you can see your updates locally.

File: data/events/2017-philadelphia.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+year: "2017" # The year of the event. Make sure it is in quotes.
+city: "Philadelphia" # The displayed city name of the event. Capitalize it.
+event_twitter: "DevOpsDaysPHL" # The twitter handle for your event, which is often devopsdays and then an abbreviation for the city
+description: # A short blurb of text to describe your event, defaults to common DevOpsDays description
+
+# All dates are in unquoted 2017-MM-DD, like this:   variable: 2016-01-05
+startdate: 2017-10-24 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2017-10-25 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: TBA # start accepting talk proposals.
```
Comment (phrawzty) [inline data/events/2017-philadelphia.yml]: All date fields need to be in the format `YYYY-MM-DD` (see line 7). The string `TBA` will break the site generator [like this](https://travis-ci.org/devopsdays/devopsdays-web/builds/206227091):
```
ERROR 2017/02/28 16:00:30 theme/partials/cfp.html template: theme/partials/cfp.html:9:14: executing "theme/partials/cfp.html" at <dateFormat "2006-01-...>: error calling dateFormat: Could not parse Date/Time format: Unable to parse date: TBA
```

**Comment by bridgetkromhout** [comment]
I went ahead and fixed it.

**Comment by peterjshan** [comment]
@bridgetkromhout THANK YOU!

**Pull request merged**